src/Hooks/ControlValuesContext.js

```diff
@@ -17,13 +17,13 @@ export const getValueAndStatus = (ctrlVals, document, field) => ({
 Consists of three parts:
  * ctrlVals: an object representing the control values (see above)
  * dispatchCtrlVals: the function that a control component should call to update the
-   ctrlVals object; automatically handles detecting when a local value has change to be equivalent
-   to the corresponding remote value.
+	 ctrlVals object; automatically handles detecting when a local value has change to be equivalent
+	 to the corresponding remote value.
  * reloadRemoteValues: a function that reloads the remote values (duh). Takes a boolean for whether
-   the local values should be discarded too.
+	 the local values should be discarded too.
 */
 export const ControlValuesContext = createContext({
-	ctrlVals: { remote: null, local: {} },
+	ctrlVals: { remote: {}, local: {} },
 	dispatchCtrlVals: () => {},
 	reloadRemoteValues: () => {},
```

Comment:
I think this function can be deleted since it's not used anywhere and doesn't make sense in our framework of "remote values are always in sync anyway."  Should also be deleted from ControlPanel.jsx:54.